sdk/test/trace/probability_sampler_test.cc

```diff
@@ -166,7 +170,8 @@ TEST(ProbabilitySampler, ProbabilitySamplerOnePercent)
 TEST(ProbabilitySampler, ProbabilitySamplerAll)
 {
   double probability = 1.0;
-  int iterations = 100000, expected_count = iterations * probability;
+  int iterations     = 100000;
```

Comment:
My only comment here is that the TCs for the probability sampler are very repetitive, containing the same identical code. We could have moved the body into a template method, use constexpr for the percentage parameter, and shrunk the code x10 times. Less LOC is the best.. Maybe even collapsing all into for each - percentage, run, print result, all into just one test that prints a failure if either of the subtests fail.

Reply:
This is good suggestion, opened issue https://github.com/open-telemetry/opentelemetry-cpp/issues/350 for tracking.